package com.kuark.tool.advance.advance20191005;

/**
 * @author rock
 * @detail
 * @date 2019/10/5 16:01
 */
public class ExceptionTest {

    public static void main(String[] a){
        //也是继承自Exception，为什么不用throws，应该是每个方法都会默认有个throws
        //这个异常，会一直往上抛
//        RuntimeException
    }
}
